sdk/tables/azure-data-tables/samples/async_samples/sample_create_client_async.py

```diff
@@ -49,6 +49,7 @@ async def create_table_client(self):
         )
         print("Table name: {}".format(table_client.table_name))
         # [END create_table_client]
+        await table_client.close()
```

Comment:
Rather than call explicit close, we should create the client in an async with statement:
```python
async with TableServiceClient() as service_client:
    ...
```